Events/MessageReceivedEvent.cs

```diff
@@ -12,6 +12,7 @@ public class MessageReceivedEvent : Event
         public const string NAME = "Message received";
         public const string DESCRIPTION = "Triggered when you receive a message";
         public const string SAMPLE = "{\"timestamp\":\"2016-06-10T14:32:03Z\",\"event\":\"ReceiveText\",\"From\":\"$cmdr_decorate:#name=HRC1;\",\"Message\":\"Hello\"}";
+        //public const string SAMPLE = "{\"timestamp\":\"2017-10-01T18:51:52Z\", \"event\":\"ReceiveText\", \"From\":\"Minoo Minou\", \"Message\":\"u're welcome\", \"Channel\":\"player\"}";
```

Comment:
Please don't leave "dead code" commented out in the code base because it will cause every future dev, including future you, to waste time puzzling out what it was for and why it was left in. 

We have version control for getting the old code back if necessary.

An exception is if you need the code only occasionally, e.g. for debugging an edge case. In that case explain with a comment why it is usually comented out.